Snake.js

```diff
@@ -199,32 +196,27 @@ Snake.prototype.createGeometry = function( geometry  , numOf ){
       normals[ index * 3  + 6 ] = n3.x;
       normals[ index * 3  + 7 ] = n3.y;
       normals[ index * 3  + 8 ] = n3.z;
-    
+   
+    } 
+        
```

Comment:
And second for the instanced data. This way the data doesn't need to be repeated for every vertex; the GPU combines the two and essentially does the inner for loop.